corehq/apps/app_manager/templates/app_manager/partials/case_list.html

```diff
@@ -30,15 +30,17 @@
         <option value="no">Don't Use Case Tiles</option>
         <option value="yes">Use Case Tiles</option>
     </select>
-    <label class="checkbox" data-bind="visible: useCaseTiles() == 'yes'">
-        <input type="checkbox" data-bind="checked: persistTileOnForms">
-        Use this case list tile persistently in forms
-    </label>
-    <label class="checkbox" data-bind="visible: useCaseTiles() == 'yes' && persistTileOnForms()">
-        <input type="checkbox" data-bind="checked: enableTilePullDown">
-        Embed case details in case tile pull-down
-    </label>
     {% endif %}
+    <div data-bind="visible: useCaseTiles() == 'yes' || window.toggles.CASE_LIST_CUSTOM_XML">
```

Comment:
Ha, now it seems that they would have to have `CASE_LIST_TILE` toggle enabled even if they were just using custom case list XML. Sorry, I think it may have been right before and I misread it.